### ISSUE TITLE
Meeting agenda with Council on 12.12.2021

### ISSUE THREAD

**Issue by geastwood** [litentry/litentry-app#656]
* [ ] Introduction about the app, walkthrough
* [ ] Talk about the challenges we had, especially around wallet integration, and explain our rationale for integrating wallet functions
* [ ] collect Feedback
* [ ] discuss next step

Meeting duration: 1 hour

FYI: @litentry/rn-team

**Comment by stephanie-olp** [comment]
Summary of the meeting with Raul:
* We gave him a **demo** of the app with all the different features: He really liked the design of the app
* Raul warned us about the high possibility of a **reshaping of the governance collective** with the **dissolution of the council** The goal would be to have everything voted through democracy. That should happen in **Q2/2022**
* We planned on opening an **element channel with the parity team** so they can give us feedback on the app and updates about the ecosystem
* He questioned us on the **delivery of Milestone 2 and 3:** We answered that the features of the Milestone 2 were ready and that more than half of the Milestone 3 was done too. We explained that the delay was mostly due to some dependence to other projects and changes of direction compared to the original proposal. He told us to prepare the **draft of Milestone 3** by **end of January 2022** and share with them the changes we have done from the original proposal and the reasons why.
* He asked us to add him to our internal testing channel 
* He told us to open public testing as soon as possible: We told him we needed some time to prepare some marketing material, and we wanted to do it **after the Christmas holidays** so the community will be fully engaged.
